taxonomies/labels.txt

```diff
@@ -39,11 +43,13 @@ synonyms:es:GMS, glutamato, glutamato monosódico
 
 synonyms:en:preservatives, conservatives
 synonyms:fi:säilöntäaineita
+synonyms:fr:conservateurs
```

Comment:
Synonyms entries are needed only to define synonyms: A is equal to B. Here "conservateurs" does not have a synonym.

Reply:
What do you mean?

Reply:
synonyms:en:preservatives, conservatives  --> makes conservatives a synonym of preservatives in labels. e.g. "without conservatives" will match "without preservatives".

synonyms:fr:conservateurs --> doesn't do anything (and will probably break the taxonomy building). It should be removed.